src/test/java/com/celements/course/service/CourseServiceTestModelAccessMock.java

```diff
@@ -0,0 +1,127 @@
+package com.celements.course.service;
+
+import static com.celements.common.test.CelementsTestUtils.*;
+import static org.easymock.EasyMock.*;
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.velocity.VelocityContext;
+import org.junit.Before;
+import org.junit.Test;
+import org.xwiki.model.reference.DocumentReference;
+
+import com.celements.common.test.AbstractComponentTest;
+import com.celements.course.classcollections.CourseClasses;
+import com.celements.model.access.IModelAccessFacade;
+import com.celements.rendering.RenderCommand;
+import com.xpn.xwiki.objects.BaseObject;
+import com.xpn.xwiki.web.Utils;
+
+public class CourseServiceTestModelAccessMock extends AbstractComponentTest {
```

Comment:
names of junit test classes MUST end on *Test. Otherwise they are not executed in maven context!